doc/services/portability/posix/option_groups/index.rst

```diff
@@ -603,18 +603,22 @@ Additional POSIX Options
 _POSIX_ASYNCHRONOUS_IO
 ++++++++++++++++++++++
 
+Functions part of the ``_POSIX_ASYNCHRONOUS_IO`` Option are not implemented in Zephyr but are
+provided so that conformant applications can still link. These functions will fail setting
```

Comment:
Will sound like a nit but this missing comma makes a big difference -- WDYT?

```suggestion
provided so that conformant applications can still link. These functions will fail, setting
```

Reply:
Fixed!